### ISSUE TITLE
'Share' button is blocked by Ad Blocker

### ISSUE THREAD

**Issue by huming2207** [filebrowser/filebrowser#632]
**Expected behaviour**
There is a "Share" button on the top right corner, where I can generate a share link. On Windows, it works fine, but not on Linux

**What is happening instead?**
Check this out:
![image](https://user-images.githubusercontent.com/4463497/51022533-99dc9480-15d8-11e9-8c43-c7dafb54318d.png)

**Additional context**
My current environment is:
- Manjaro Linux 18.0.1
- Chromium Browser, Version 71.0.3578.98

Same results on Ubuntu 18.04 with Google Chrome.

**How to reproduce?**
1. Grab a Linux distro installed
2. Open a certain directory, select a certain file
3. No share button shows up

**Comment by hacdias** [comment]
Hello @huming2207! Please check your user settings, please. If you're using the same installation with the exact same user, please tell me if you see anything odd in the Developers Console (F12) or in the server logs.

**Comment by huming2207** [comment]
Hi @hacdias , 

Thanks for your reply. I've just found on my another laptop also have the same problem. It's running Windows 10 17763.253 and Google Chrome 71.0.3578.98. Then I cleared up the cache and turned off Adblock Plus, then the share button shows up.

I guess my Adblock Plus rules are too strict, or maybe your implementation somehow triggered the Adblock Plus. 

![image](https://user-images.githubusercontent.com/4463497/51064932-5163bc00-1656-11e9-8104-2e5e01383377.png)


Regards,
Jackson

**Comment by hacdias** [comment]
oh, well... that's weird. The only difference between that button and the others is just the icon and the text. Maybe the 'share' keyword triggers AdBlock Plus, which is, in my opinion, a bit stupid. But I don't believe I can do anything about it. It's still the best word and icon.

What do you think?

**Comment by huming2207** [comment]
Yea, I guess we might either document this issue and tell the user to either disable Adblock Plus for their FileBrowser sites, or disable the additional "acceptable ads" and "block social media tracking" functions. Or alternatively, we might rename the share button to "Generate sharable download link" though it sounds weird...

**Comment by hacdias** [comment]
@huming2207 I think it's probably the social icons blocking. This icon is called 'share' or something like that (the actual button on the icons font). So it may be getting blocked because of that. Unfortunately, I can't do anything to prevent the ad-blocker not to block the icon.

I'm closing this, and updating the title if someone in the future runs in the same issue. I'm also setting this as 'Out of scope' because the resolution of this issue is out of our scope.

Icon reference: https://material.io/tools/icons/?search=share&icon=share&style=baseline

If you find a way to solve this without needing to change the button, nor its text, a PR would be really welcome on filebrowser/frontend.
